runtime/dart_isolate.h

```diff
@@ -99,7 +100,8 @@ class DartIsolate : public UIDartState {
     ///
     Unknown,
     //--------------------------------------------------------------------------
-    /// The Dart isolate has been created but none of the library tag or message
+    /// The Dart isolate has been created but none of the library tag or
```

Comment:
Clang format attacked this on me. I'll see if it's possible to revert, but it may just be a new version of clang-format running on a file that hasn't been touched in a while.

Reply:
Done.